plugins/BEdita/API/tests/TestCase/Event/CommonEventHandlerTest.php

```diff
@@ -13,6 +13,7 @@
 namespace BEdita\API\Test\Event;
```

Comment:
(Not part of this PR) Namespace is wrong: `BEdita\API\Test\TestCase\Event`.

Also, these imports are unnecessary:
```
use Cake\Network\Exception\UnauthorizedException;
use Cake\ORM\TableRegistry;
```